### DISCUSSION TITLE
jpegli support

### DISCUSSION THREAD

**Discussion by adityapatadia** [libvips/libvips#3871] [Ideas]
I just read that libjxl has a library which backports the advantages of jpeg-xl to good old JPEG. The library is called jpegli. Some information is available in this comment: https://news.ycombinator.com/item?id=38644005

I know it's just a very bird eye view but we should plan to introduce this in VIPS. May be we can generate better JPEGs.

**Comment by kurtextrem** [comment]
I've added also a few quotes here which highlight why jpegli might be of high interest: https://github.com/lovell/sharp/issues/4018

**Comment by jcupitt** [comment]
I agree, this sounds great. Let's make an enhancement issue and continue any discussion there.

**Comment by jcupitt** [comment]
Enhancement issue here: https://github.com/libvips/libvips/issues/3872

**Comment by erickguan** [comment]
A few articles I skimmed through:
- https://cloudinary.com/blog/jpeg-xl-and-the-pareto-front
- https://giannirosato.com/blog/post/jpegli/
The second blog has a few benchmark with commands.

**Comment by jcupitt** [comment]
Let's move discussion to the enhancement issue.
